client/src/components/App.js

```diff
@@ -1,17 +1,27 @@
 import React from 'react';
+import { connect } from 'react-redux';
 import 'primereact/resources/themes/saga-blue/theme.css';
 import 'primereact/resources/primereact.min.css';
 import 'primeflex/primeflex.css';
 import 'primeicons/primeicons.css';
-import { Lists } from './Lists';
+import Lists from './Lists';
```

Comment:
Don't use export default.

Reply:
I used export default because export const wouldn't work for some reason when I tried exporting the Redux connected component. I can try again. 

Otherwise, I always avoided export default like you said